graylog2-server/src/main/java/org/graylog/events/contentpack/entities/EmailEventNotificationConfigEntity.java

```diff
@@ -85,6 +89,9 @@ public static Builder create() {
         @JsonProperty(FIELD_SENDER)
         public abstract Builder sender(ValueReference sender);
 
+        @JsonProperty(FIELD_REPLY_TO)
+        public abstract Builder replyTo(ValueReference sender);
```

Comment:
Because `replyTo` is not optional, we need to add a default in the `create()` method in line `82` in order to still support the installation of older content packs which don't have the property set.

Reply:
Good catch - added that